CHANGELOG

```diff
@@ -0,0 +1,107 @@
+==== 2.1 beta 2017-12-22 ====
+After two years of hard work, we are pleased to announce Mixxx 2.1 beta. We
+have overhauled the effects system, redesigned the skins, added and improved
+lots of controller mappings, rewrote the audio file decoders twice, and of
+course fixed a bunch of bugs. Download it!
+
+Here is a quick summary of what is new in Mixxx 2.1:
+  * Skins scale for high resolution screens
+  * Overhauled Deere and LateNight skins
+  * New Tango skin
+  * Effects are synchronized to the tempo
+  * Effects are processed post-fader and post-crossfader and can be previewed
+in headphones
+  * One metaknob per effect for powerful control of effect chains
+  * Nine new effects: Autopan, Biquad Equalizer, Biquad Full Kill Equalizer,
+Loudness Contour, Metronome, Parametric Equalizer, Phaser, Stereo Balance,
+Tremolo
+  * Loaded effects are saved and restored when Mixxx restarts
```

Comment:
Loaded effects and each effect's settings ...

Reply:
How about "Loaded effects and their parameters are saved"?